server.js

```diff
@@ -0,0 +1,15 @@
+const jsonServer = require('json-server');
+const server = jsonServer.create();
+const router = jsonServer.router('db.json');
+const middlewares = jsonServer.defaults();
+
+server.use(
+  jsonServer.rewriter({
+    '/api/*': '/$1',
+  })
+);
+server.use(middlewares);
+server.use(router);
+server.listen(8080, () => {
+  console.log('JSON Server is running');
+});
```

Comment:
I think you have an existing `index.js` with express. 
If you like to load json-server in your server, you have to combine these files.

Reply:
Thank you 👍🏼